### DISCUSSION TITLE
raucb file header

### DISCUSSION THREAD

**Discussion by liuruan** [rauc/rauc#898] [Q&A]
Where can I find the header information of the raucb file format?

**Comment by gajanak** [comment]
It is a squashfs Image with some Crypto Data Suffix.
Perhaps this helps....

**Comment by ejoerns** [comment]
There are different variants of formats, these are documented here:

https://rauc.readthedocs.io/en/latest/reference.html#sec-ref-formats

@liuruan may I know the background/intention of your question? Maybe then we can provide more targeted info/help
